### PR TITLE
Staging/master sync

### PR DESCRIPTION
**Pull request by llewellyn-sl**

### CODE REVIEW

**Review by swampie — DISMISSED** [review]

File: docs/wave.md [hunk 1]
```diff
@@ -2,19 +2,23 @@
...
 - The Tower installation should allow ingress traffic from the Wave service
-- The `TOWER_ENABLE_WAVE` environment variable must be set to `true`, or the `enable_wave` parameter set to `true` in your tower.yml configuration
 
-When these conditions are met, the Wave feature is available on the Tower compute environment creation page (available for AWS compute environments only)
+- The `TOWER_ENABLE_WAVE` environment variable must be set to `true` in Tower configuration environment
+
+When these conditions are met, the Wave feature is available on the Tower compute environment creation page (currently only available for AWS compute environments).
 
-Once Wave is enabled, a secure channel is established between Tower and the Wave service to exchange encrypted data securely.
+Once Wave is enabled, it will be possible to use private container repositories and the Fusion file system in your Nextflow pipelines.
```
[the remark this answers is not in the document]
Comment (pditommaso) [reply docs/wave.md:22]: Was not agree to move this only into install.tower.nf?

[the remark this answers is not in the document]
Comment (llewellyn-sl) [reply docs/wave.md:22]: @swampie @pditommaso This is the old version of wave.md, no longer present in the mkdocs.yml for help.tower.nf. Will remove the file entirely.

[the remark this answers is not in the document]
Comment (llewellyn-sl) [reply docs/wave.md]: @swampie would you suggest a note explicitly stating that private certificates are not yet supported for Wave? (for the install docs version of wave.md)

[the remark this answers is not in the document]
Comment (swampie) [reply docs/wave.md]: Yes: I would state this explicitely until we don't have a solution as we have customers that have higher expectations and bumped hardly into this problem. @pditommaso wdyt?

[the remark this answers is not in the document]
Comment (pditommaso) [reply docs/wave.md]: Yes, better specifying it

[the remark this answers is not in the document]
Comment (llewellyn-sl) [reply docs/wave.md]: Will add a note to the Wave page in Install docs.

**Review by swampie — APPROVED** [review]

**Pull request merged**